requirements/app/ui.txt

```diff
@@ -1 +1,3 @@
 streamlit>=1.3.1, <=1.11.1
+panel
+param
```

Comment:
Can we get some version ranges?

Reply:
panel>=0.12
param>=1.9.0

Param is a dependency of Panel.

Reply:
Since param is a dep of panel, then we can exclude it here in the list. I chose `panel>=0.12, <=0.13.1` since 0.13.1 is the latest currently available :)